apply_all.bat

```diff
@@ -8,7 +8,7 @@
 set StartAndWait=start /min /wait
 set SqlCmd=sqlcmd -E -S %computername%\PERPSQL -d perpetuumsa -I -i
 
-set TARGET_DIR=C:\PerpetuumServer\data
+set TARGET_DIR=H:\PerpetuumServer\data
```

Comment:
When running the script to test the PR, I need to change it back to drive C to make it work.
What could be the solution to prevent breaking change when deploying/developing?

Reply:
It's my personal settings accidently commited, fixed